tutorials/international/index.md

```diff
@@ -25,6 +25,56 @@ supports. Flutter provides widgets and classes that help with
 internationalization and the Flutter libraries themselves are
 internationalized.
 
+### Setting up an Internationalized app: the `flutter_localizations` package
+
+By default Flutter only provides US English localizations. To add
+support for other languages, applications must specify some additional
+`MaterialApp` properties and include a separate package called
+`flutter_localizations`.  This package currently supports about 15
+languages with more to come.
+
+International apps must add a `pubspec.yaml` dependency on flutter_localizations:
+
+```dart
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_localizations:
+    sdk: flutter
+```
+
+International apps must also add an import for flutter_localizations
+and specify `localizationsDelegates` and `supportedLocales` for MaterialApp:
+
+```dart
+import 'package:flutter_localizations/flutter_localizations.dart';
+
+new MaterialApp(
+ localizationsDelegates: [
+   // ... app-specific localization delegate[s] here
+   GlobalMaterialLocalizations.delegate,
+   GlobalWidgetsLocalizations.delegate,
+ ],
+ supportedLocales: [
+    const Locale('en', 'US'), // English
+    const Locale('he', 'IL'), // Hebrew
+    // ... other locales the app supports
+  ],
+  // ...
+)
+```
+
+The elements of `localizationsDelegates` are factories that produce
```

Comment:
what do you mean "elements" here?

classes?

Reply:
localizationsDelegates is a List, so list elements.